FlowCrypt/Controllers/SideMenu/Menu/MyMenuViewController.swift

```diff
@@ -335,6 +341,8 @@ extension MyMenuViewController {
                     showAlert(message: error.errorMessage)
                 }
             }
+        default:
+            break
```

Comment:
let's use `case .separator` instead of `default`, so it'll show warning when another folder type will be added